src/akkudoktoreos/devices/battery.py

```diff
@@ -10,345 +10,273 @@
 logger = get_logger(__name__)
 
 
-def max_ladeleistung_w_field(default: Optional[float] = None) -> Optional[float]:
+def max_charging_power_field(description: Optional[str] = "") -> float:
     return Field(
-        default=default,
+        default=10000,
         gt=0,
-        description="An integer representing the charging power of the battery in watts.",
+        description=description,
     )
 
 
-def start_soc_prozent_field(description: str) -> int:
+def initial_soc_percentage_field(description: str) -> int:
     return Field(default=0, ge=0, le=100, description=description)
 
 
-class BaseAkkuParameters(BaseModel):
-    kapazitaet_wh: int = Field(
+class BaseBatteryParameters(BaseModel):
+    """Base class for battery parameters with fields for capacity, efficiency, and state of charge."""
+
+    capacity_wh: int = Field(
         gt=0, description="An integer representing the capacity of the battery in watt-hours."
     )
-    lade_effizienz: float = Field(
+    charging_efficiency: float = Field(
         default=0.88,
         gt=0,
         le=1,
         description="A float representing the charging efficiency of the battery.",
     )
-    entlade_effizienz: float = Field(default=0.88, gt=0, le=1)
-    max_ladeleistung_w: Optional[float] = max_ladeleistung_w_field()
-    start_soc_prozent: int = start_soc_prozent_field(
+    discharging_efficiency: float = Field(
+        default=0.88,
+        gt=0,
+        le=1,
+        description="A float representing the discharge efficiency of the battery.",
+    )
+    max_charge_power_w: Optional[float] = max_charging_power_field()
+    initial_soc_percentage: int = initial_soc_percentage_field(
         "An integer representing the state of charge of the battery at the **start** of the current hour (not the current state)."
     )
-    min_soc_prozent: int = Field(
+    min_soc_percentage: int = Field(
         default=0,
         ge=0,
         le=100,
         description="An integer representing the minimum state of charge (SOC) of the battery in percentage.",
     )
-    max_soc_prozent: int = Field(default=100, ge=0, le=100)
+    max_soc_percentage: int = Field(
+        default=100,
+        ge=0,
+        le=100,
+        description="An integer representing the maximum state of charge (SOC) of the battery in percentage.",
+    )
 
 
-class PVAkkuParameters(BaseAkkuParameters):
-    max_ladeleistung_w: Optional[float] = max_ladeleistung_w_field(5000)
+class SolarPanelBatteryParameters(BaseBatteryParameters):
+    max_charge_power_w: Optional[float] = max_charging_power_field()
 
 
-class EAutoParameters(BaseAkkuParameters):
-    entlade_effizienz: float = 1.0
-    start_soc_prozent: int = start_soc_prozent_field(
+class ElectricVehicleParameters(BaseBatteryParameters):
+    """Parameters specific to an electric vehicle (EV)."""
+
+    discharging_efficiency: float = 1.0
```

Comment:
this will probably also overwrite any description, so should use a field helper method with different default value

Reply:
lets address that in a new PR please